### PR TITLE
Tidy Up - Nothing Deleted, Just Sorted

### PR DESCRIPTION
**Pull request by Jakosaur**
Just merged sections inside of sets and added more Season numbers. 👍

### CODE REVIEW

File: src/setData/yearOne.js [hunk 1]
```diff
@@ -536,13 +513,7 @@ export default ([
             name: 'Emblems',
```
Comment (joshhunt) [inline src/setData/yearOne.js]: Could you change this to "Extras" pelase?

Comment (Jakosaur) [reply src/setData/yearOne.js]: Can't believe I missed one out of them all haha

**Comment by joshhunt** [comment]
thanks a lot for this! I merged everything together rather quickly on the train, with the intention on cleaning it up a bit later. thanks for sorting this out ✨

**Comment by Jakosaur** [comment]
NP, you did an excellent job merging them together 👍

**Pull request merged**